src/components/inline-link/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactElement } from 'react'
+import classNames from 'classnames'
+import MaybeInternalLink from 'components/maybe-internal-link'
+import Text from 'components/text'
+import { InlineLinkProps } from './types'
+import s from './inline-link.module.css'
+
+const InlineLink = ({
+  className,
+  href,
+  text,
+  textSize,
+  textWeight,
+}: InlineLinkProps): ReactElement => {
+  const classes = classNames(s.root, className)
+
+  return (
+    <MaybeInternalLink className={classes} href={href}>
+      <Text asElement="span" size={textSize} weight={textWeight}>
```

Comment:
this, according to the design specs, is not entirely correct. in Figma the `LinkStandalone` component is not using the `Body100/200/300` styles because the height of the bounding-box of the component should match the height of the icon for that specific `LinkStandalone` size:

| Size | Bounding-box | Icon | Text (font-size) | Text (line-height) |
|---|:---:|---:|---:|---:|
| small | 16px |  16px | 13px | 16px |
| medium | 16px |  16px | 14px | 16px |
| large | 24px |  24px | 16px | 24px |

While if we use the standard `body` styles we have:
| Size | Text (font-size) | Text (line-height) | |
|---|:---:|---:|---:|
| body-100 | 13px | 18px | ❌ |
| body-200 | 14px |  20px | ❌ |
| body-300 | 16px | 24px | ✅ |

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/100/commits/6345c49eadd48235e7170a73eb19d890cd622232, also added updated a Swingset example & added an extra comment in https://github.com/hashicorp/dev-portal/pull/100/commits/e09c42862f98ae7a1a90685702d7220fa0806be6.